src/SuccessModal.jsx

```diff
@@ -3,6 +3,19 @@ import { closeIcon } from "./icons";
 
 export default function SuccessModal({ isVisible, setIsVisible }) {
 	if (!isVisible) return null;
+	useEffect(() => {
```

Comment:
I get an error following upload success saying useEffect is not defined - I presume it needs importing at the top. Did it work for you though?

Reply:
it did work for me..weird

Reply:
Working now